cachito/web/static/api_v1.yaml

```
@@ -146,6 +146,30 @@ paths:
             type: string
             example: gomod
             default: null
+        - name: created_from
+          in: query
+          description: >
+            Filter requests by the given date or datetime. This
+            argument filters all requests created after the specified
+            date or datetime.
+          schema:
+            type: string
+            example:
+              date: "2021-10-07"
+              datetime: "2021-10-07T18:52:57.577851"
+            default: null
+        - name: created_to
+          in: query
+          description: >
+            Filter requests by the given date or datetime. This
```

Do you think it's good idea to point that using date here is inclusive?

```suggestion
            Filter requests by the given date (inclusive) or datetime. This
```